app/i18n/de-CH.json

```diff
@@ -85,6 +85,9 @@
   "startup.auth.mfaCode": "Code",
   "startup.auth.mfaDescription": "Dieses Konto ist mit multi factor authentication geschützt. Bitte gib den Code vom google authenticator ein:",
   "startup.auth.message": "Melde dich bei deinem balloon an:",
+  "startup.selective.continue": "Weiter",
+  "startup.selective.edit": "Selektion anpassen",
+  "startup.selective.message": "Standarmässig werden shares bei der Synchronisation ignoriert. Du kannst die Selektion hier oder jederzeit später in den Einstellungen ändern.",
```

Comment:
Nur deine persönlichen Daten werden standarmässig synchronisiert. Freigaben müssen separat aktiviert werden.
Du kannst jetzt zusätzliche Ordner aktivieren oder jederzeit später in den Einstellungen.

Hinweis: Neue Freigaben werden standardmässig nicht synchronisiert und müssen in den Einstellungen aktiviert werden.